introduction_to_amazon_algorithms/xgboost_abalone/xgboost_abalone.ipynb

```diff
@@ -2,7 +2,9 @@
  "cells": [
```

Comment:
<blockquote>To create a tuning job using the AWS SageMaker Automatic Model Tuning API, you need to define 3 attributes.  </blockquote><p>"three parameters" instead "3 attributes." </p><p> </p><blockquote>(to specify settings...)</blockquote><p>"(which specifies settings..." </p><p> </p><blockquote>(to configure the...</blockquote><p>"(which configures the..."</p><p> </p><blockquote>To learn more about that, </blockquote><p>"these" instead of "that"</p><p> </p><p>These are nits, feel free to ignore. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3623/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>